docs/docs/components/api/README.md

```diff
@@ -736,6 +736,18 @@ async run({ $ }) {
 
 It functions the same way as [`$end` in workflow code steps](/workflows/steps/code/#end).
 
+**`$.summary`**
+
+`$.summary` is used to surface brief, user-friendly summaries about what happened when an action step succeeds. For example, when [adding items to a Spotify playlist](https://github.com/PipedreamHQ/pipedream/blob/master/components/spotify/actions/add-items-to-playlist/add-items-to-playlist.mjs#L51):
+<div>
+<img alt="Spotify example with $summary" src="./images/spotify-$summary-example.png">
+</div>
+
+Example implementation:
+```
+$.export("$summary", `Successfully added ${data.uris.length} ${data.uris.length == 1 ? "item" : "items"} to "${get(playlistId, "label", playlistId)}"`);
```

Comment:
Sorry I noticed one more small thing - you're using `lodash/get` in this example, so it won't work as written. I would recommend fixing that and also fixing sample variables so they can run the full example:

```javascript
const data = [1, 2]
const playlistId = "Cool jams"
$.export("$summary", `Successfully added ${data.length} ${data.length == 1 ? "item" : "items"} to ${playlistId}`);
```

Reply:
wait @dylburger I don't understand. I copied this from the published code and it works in prod:
https://pipedream.com/apps/spotify/actions/add-item-to-a-playlist

Reply:
Clarified via Slack!